src/SingleProject/Resizetizer/src/DetectInvalidResourceOutputFilenamesTask.cs

```diff
@@ -36,7 +36,7 @@ public override bool Execute()
 						{
 							var filename = item.ItemSpec;
 
-							if (!Utils.IsValidResourceFilename(filename) || !File.Exists(filename))
```

Comment:
If the file does not exist, it doesn't matter as the next task will throw. We are focused on the name not the existence.